.changeset/tender-weeks-help.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-api-docs': major
```

Comment:
This should be a patch release for this :pray:

Reply:
Okay